src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,39 @@
+import { NavLink, useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+
+export const Navbar = () => {
+  const [searchParams] = useSearchParams();
+  const getLinkClass = ({ isActive }: { isActive: boolean }) => cn(
+    'navbar-item',
+    { 'has-background-grey-lighter': isActive },
+  );
```

Comment:
It is better to define this function outside the component so that it is not created every time a new render is performed.